src/Faithlife.Tracing.AspNetMvc/AspNetMvcTracing.cs

```diff
@@ -20,6 +20,6 @@ public static void Initialize(HttpApplication application, AspNetMvcTracingSetti
 			}
 		}
 
-		public static ITraceProvider GetProvider(HttpContext httpContext) => AspNetTracing.GetProvider(httpContext);
+		public static ITraceProvider GetProvider(HttpContext httpContext) => AspNetTracing.GetProvider(httpContext) ?? NullTraceProvider.Instance;
```

Comment:
Should we let the client decide whether to use a `NullTraceProvider` as an alternative?

Reply:
That's fair.